docs/backends/AWSBatch.md

```diff
@@ -0,0 +1,77 @@
+# AWS Batch Backend
+
+AWS Batch is a set of batch management capabilities that dynamically provision the optimal quantity and type of compute resources (e.g., CPU or memory optimized instances) based on the volume and specific resource requirements of the batch jobs submitted.
+
+This section provides details on how to configure the AWS Batch backend with Cromwell. For instructions on common configuration and deployment tutorial, see [Getting started with AWS Batch](https://cromwell.readthedocs.io/en/develop/tutorials/AwsBatch101/). 
+
+
+
+## Resources and Runtime Attributes
+
+Cromwell and AWS Batch recognizes number of runtime attributes, more information can be found in the [customize tasks](/RuntimeAttributes#recognized-runtime-attributes-and-backends) page.
+
+
+## Running Cromwell on an EC2 instance
+
+Cromwell can be run on an EC2 instance and submit jobs to AWS Batch, AWS provide [CloudFormation stacks and guides](https://docs.opendata.aws/genomics-workflows/orchestration/cromwell/cromwell-overview/) to building the correct IAM permissions.
+
+
+
+## Scaling Requirements
+For a Cromwell server that will run multiple workflows, or workflows with many steps (e.g. ones with large scatter steps), it is recommended to setup a database to store workflow metadata.  The application config file will expect a SQL database location. Follow [these instructions](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-serverless.create.html) on how to create a serverless Amazon Aurora database. 
+
+## Configuring Cromwell for AWS Batch
+
+Within the `*.conf` file, you have a number of options to change the Cromwell's interaction with AWS Batch.
+
+### Filesystems
+> More information about filesystems can be found on the [Filesystems page](/filesystems/Filesystems/).
+> 
+
+Amazon's S3 storage is a supported filesystem in both the engine and backend, this means that S3 files can be referenced at a workflow level, and as input files, provided they are prefixed by `'s3://'`.
+
+* filesystems
+* filesystems.s3.auth
+* filesystems.s3.caching.duplication-strategy
+
+### Configuring Authentication
+
+To allow Cromwell to talk to AWS, we have a few authentication schemes that we can specify in the configuration that Cromwell can accept:
+-  `default`: This uses the [default authentication provider](https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html), which uses the standard AWS search paths:
+    - Environment Variables - `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`
```

Comment:
I haven't tried it myself but the word on the street is that only `default` is properly working at the moment

Reply:
Yeah I haven't tried them before, just pulling them from the previous docs. Should I comment them out for now?

Reply:
I think so

Reply:
Rewritten (f2778a99fec0127e843b7b6a25b89a3edd9974ad) to read:

> To allow Cromwell to talk to AWS, the `default` authentication scheme uses the [default authentication provider](https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html) with the following AWS search paths:
> - Environment Variables - `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`
> - Java system properties - `aws.accessKeyId` and `aws.secretKey`
> - Default credential profiles file - Created by the AWS CLI, typically located at `~/.aws/credentials`
> - _Instance profile credentials_ - Only relevant on EC2 instances